flow/connectors/postgres/qvalue_convert.go

```diff
@@ -387,7 +387,7 @@ func parseFieldFromQValueKind(qvalueKind qvalue.QValueKind, value any) (qvalue.Q
 		timeVal = strings.Replace(timeVal, "24:00:00.000000", "23:59:59.999999", 1)
 		// edge case, Postgres prints +0000 as +00
 		timeVal = strings.Replace(timeVal, "+00", "+0000", 1)
```

Comment:
this will break if postgres sends `+0000` ever, ideally it'd check for `\+00([^\d]|$)`

Reply:
There are some records like this in my table

INSERT INTO public.workers_schedule (id, start_time, end_time, worker_id, week_day) VALUES (5919, '08:00:00 +00:00', '21:00:00 +00:00', '2c3e2124-3ca1-487f-b4ca-543874577c67', 'TH');

Reply:
Good to know, tho this code only needs to accept what postgres would output, rather than what it accepts as input

Will address this gap still